.circleci/config.yml

```diff
@@ -34,6 +34,31 @@ jobs:
             pip install .
             pip install scipy
 
+      - run:
+          name: install towncrier
+          command: |
+            . venv/bin/activate
+            cd ..
+            # Get towncrier from the main repository until next release
+            # (the current release is 19.2.0)
+            git clone --single-branch --branch master https://github.com/hawkowl/towncrier.git
+            cd towncrier
+            pip install .
```

Comment:
couldn't this use `pip install git+https://github.com/hawkowl/towncrier.git@master`

Reply:
oh, that is much nicer, was not really aware of that option.